modules/nf-core/genescopefk/main.nf

```diff
@@ -36,6 +37,8 @@ process GENESCOPEFK {
         --output . \\
         --name_prefix ${prefix}
 
+    printf -v KMERCOV "%.2f\\n" \$( grep "^kmercov" *_model.txt | cut -d" " -f2 )
```

Comment:
TIL I learned there is `printf -v` to create new environment variables. 👍🏼 

As far as I can tell, `printf -v` is a bash builtin. Are we certain this script section _always_ runs in bash ?

Reply:
It works in ksh and a few other shells too, but I can add the directive at the top.